### ISSUE TITLE
Unable to add to shipping cart

### ISSUE THREAD

**Issue by Mordalak1999** [BrickTwo/BrickHunter#182]
Hello 
I keep getting this message when I want to add to or clear my shipping cart

**Action cannot be performed!
To perform this action, the LEGO.com website must be open and fully loaded in another tab.**

But I have a tab opened to https://www.lego.com/en-ca

Any idea ?

**Comment by Mordalak1999** [comment]
I have tested in Chrome 102.2 and Edge 102.0
Both are working fine
The problem is just in Firefox

**Comment by BrickTwo** [comment]
yeah, it's weird that the browser api tells me sometimes that there is no lego tab open even though it clearly is... and it's not specific to a browser. normally it helps to reload the lego tab again, close all tabs with lego.com or if that doesn't help to restart the browser.
